home/templates/home/tags.html

```diff
@@ -18,11 +18,14 @@
                 <br><br>
                 <div class="wrapper">
                     {% for tag in tags %}
-                    <div class="card border-secondary mb-3">
-                        <div class="card-body">
-                            <p class="card-text">{{tag.tag_name}}</p>
+
+                      <div class="card border-secondary mb-3">
+                            <div class="card-body">
+                                <p class="card-text tag-text">
+                                    <a href="{% url 'explore-page' %}?tag={{tag}}">#{{tag.tag_name}}</a>
```

Comment:
maybe space this for readability -> {{ tag }} , {{ tag.tag_name }}

Reply:
@liornoy
 I don't think it really matters to readability but I added spaces next to the bracket according to your request.